src/test/generate-sym-test.py

```diff
@@ -105,9 +105,9 @@ def process_source_file(file):
         }
 
         for (j = 0; symbols_from_source[j].name; j++) {
-                struct symbol*n = bsearch(symbols_from_source+j, symbols_from_source, sizeof(symbols_from_sym)/sizeof(symbols_from_sym[0])-1, sizeof(symbols_from_sym[0]), sort_callback);
```

Comment:
Please add a second commit which changes each `symbol*n` to `symbol *n`.

Reply:
Done. Do you have a code format tool?

Reply:
People use various formatters. But this is a generator of code, so most formatters wouldn't know how to deal with it.